### PR TITLE
Swik 1944 warning of simultaneous slide editing

### PR DESCRIPTION
**Pull request by sednanref**
Hi, this PR is for the warning of the simultaneous edition in the front-end, it provides a new collection for mongo to control the people editing slides. Also, please note the index over the collection to delete documents that are 30 minutes or more in the collection, I made this to avoid extreme cases where a user is in the editing mode and suddenly closes the tab/browser, the computer is turned off, the internet connection is broken, etc.

### CODE REVIEW

**Comment by sednanref** [comment]
I just saw that the continuous integration check didn't pass, and what I can see is that it's because the package 'object-id' is missing... but it is actually in the package.json ... so I don't really know what's happening, anyone has an idea?

File: application/routes.js [hunk 1]
```diff
@@ -301,7 +304,7 @@ module.exports = function(server) {
         }).unknown(),
       },
       tags: ['api'],
-      description: 'Create a new following'
+      description: 'Create a new slide edition being performed.'
```
Comment (dpaun) [inline application/routes.js]: I don't see a reason for this change...

**Review by dpaun — APPROVED** [review]
Code changes look OK to me. I haven't tested the branch yet.

**Comment by sednanref** [comment]
@dpaun sorry, the observation you made was a mistake from my side. I already fixed it :)

**Comment by dpaun** [comment]
Ah, yes, of course. No problem.

**Review by kprist — COMMENTED** [review]
A clarification, possibly leading to some changes: I can see in your new APIs you require both a JWT token (to secure the call) and a user id. I'm not sure about the exact use of the API, but I would assume this is redundant, as when you secure with JWT you essentially require a user to be logged in. The JWT token includes the userid already, so is there a reason you need it explicitly ? Is it possible that user a would use this API providing a different user id ?

**Comment by sednanref** [comment]
hi @kprist , thanks for the observation hmmm I see... actually I don't know too much about JWT, and that's why I didn't know that it contained the user id too... how can I extract it from the header? Is there any code around the services doing this already? thanks in advance :)

**Comment by kprist** [comment]
@sednanref sure, you can check out this handler function from the deck service: https://github.com/slidewiki/deck-service/blob/c6f53b97a2f44ab3b152f47da5e6d3eb495d8c9f/application/controllers/decks.js#L35 

It's for GET /decks and we have optional JWT there like in your example as well.

**Comment by sednanref** [comment]
hi @kprist , thanks for the advice! and now it's done, please let me know if there is something else that could be improved :)

**Pull request merged**